ddmrp_include_location_final/models/stock_buffer.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2025 ForgeFlow (http://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models
+
+
+class StockBuffer(models.Model):
+    _inherit = "stock.buffer"
+
+    def _search_stock_moves_incoming(self, outside_dlt=False):
+        domain = self._search_stock_moves_incoming_domain(outside_dlt=outside_dlt)
```

Comment:
Add a warning here indicating that you are overriding this function

Reply:
done!